pubspec.yaml

```diff
@@ -17,6 +17,8 @@ dependencies:
   freezed_annotation: ^2.4.1
   ###
   rxdart: ^0.27.7
+  meta: ^1.10.0
+  uuid: ^4.2.2
```

Comment:
`dart pub publish --dry-run` complained about these not being present. I added them but we could also ignore this warning if we wanted to. Thoughts?

Reply:
@cnixbtc This is a common problem with `--dry-run`, the warnings aren't crucial for most part and there's no command option/flag to ignore it's warnings. I believe Dart team is working on a solution but haven't reached a consensus yet.

Let's keep plugin as slim as possible imo, there's no need to heed non-licensing warnings.